src/components/Settings/Settings.js

```diff
@@ -71,9 +70,84 @@ const TabPaneGeneral = (store) => {
       </span>
     );
   });
+});
+
+const LayoutSettings = observer(({ store }) => {
+  return (
+    <>
+      <Checkbox
+        checked={store.settings.bottomSidePanel}
+        onChange={() => {
+          store.settings.toggleBottomSP();
+          setTimeout(triggerResizeEvent);
+        }}
+      >
+            Move sidepanel to the bottom
+      </Checkbox>
+
+      <br />
+      <Checkbox checked={store.settings.displayLabelsByDefault} onChange={store.settings.toggleSidepanelModel}>
+            Display Labels by default in Results panel
+      </Checkbox>
+
+      <br />
+      <Checkbox
+        value="Show Annotations panel"
+        defaultChecked={store.settings.showAnnotationsPanel}
+        onChange={() => {
+          store.settings.toggleAnnotationsPanel();
+        }}
+      >
+            Show Annotations panel
+      </Checkbox>
+      <br />
+      <Checkbox
+        value="Show Predictions panel"
+        defaultChecked={store.settings.showPredictionsPanel}
+        onChange={() => {
+          store.settings.togglePredictionsPanel();
+        }}
+      >
+            Show Predictions panel
+      </Checkbox>
+
+      {/* <br/> */}
+      {/* <Checkbox */}
+      {/*   value="Show image in fullsize" */}
+      {/*   defaultChecked={store.settings.imageFullSize} */}
+      {/*   onChange={() => { */}
+      {/*     store.settings.toggleImageFS(); */}
+      {/*   }} */}
+      {/* > */}
+      {/*   Show image in fullsize */}
+      {/* </Checkbox> */}
```

Comment:
is the intent of the commented out block here to put it in at some point later? or is it meant to be gone? if not needed it maybe best to remove it

Reply:
I'm not sure what this thing is and why it's commented out. I just moved this piece of code from the main Settings component. Decided to keep it that way to avoid any bugs.